docs/en/enterprise-edition/policy-reference/supply-chain-policies/terraform-policies/ensure-terraform-module-sources-use-git-url-with-commit-hash-revision.adoc

```diff
@@ -30,7 +30,7 @@
 Terraform modules are a collection of multiple resource configuration to offer an easy way of repeatable and reusable code logic.
 The most common way is to consume them through the public Terraform registry, which are connected to a VCS, like GitHub.
 This approach is problematic, because the module versions are not immutable and the module can be changed without changing the version, which makes the code vulnerable to a Supply Chain Attack.
-Therefore it is recommended to leverage Git URLs with a commit hash revision to guarantee immutability and consistency.
+Therefore it is recommended to leverage Git URLs with a commit hash revision to guarantee immutability and consistency. This is a more restrictive policy than https://docs.prismacloud.io/en/enterprise-edition/policy-reference/supply-chain-policies/terraform-policies/ensure-terraform-module-sources-use-git-url-with-commit-hash-revision[Terraform module sources do not use a git url with a tag or commit hash revision].
```

Comment:
To mitigate this risk, it is recommended to use Git URLs with a commit hash revision to guarantee immutability and consistency.